lib/ReactViews/Analytics/PointParameterEditor.jsx

```diff
@@ -20,61 +18,45 @@ const PointParameterEditor = React.createClass({
         viewState: React.PropTypes.object
     },
 
+    componentWillMount() {
+        if (!defined(this.pointParameterEditorCore)) {
+            this.pointParameterEditorCore = new PointParameterEditorCore(this.props.previewed,
```

Comment:
I should say, it very well may work in given how we're using this component currently.. but it's still not a reliable thing to do.